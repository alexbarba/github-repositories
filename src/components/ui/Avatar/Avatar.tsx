import { FC } from 'react'

type Props = {
  src: string
  className?: string
}

const Avatar: FC<Props> = ({ src, className = '' }) => {
  return (
    <figure className={className}>
      <a href={src}>
        <img
          src={src}
          className="w-full h-auto rounded-full"
          alt="avatar"
          width="296"
          height="296"
        />
      </a>
    </figure>
  )
}

export default Avatar
