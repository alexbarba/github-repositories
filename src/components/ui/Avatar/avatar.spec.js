import { render } from '@testing-library/react'
import Avatar from './Avatar'

describe('Avatar', () => {
  it('should be match with snapshot', () => {
    const { asFragment } = render(<Avatar />)
    expect(asFragment()).toMatchSnapshot()
  })
})
