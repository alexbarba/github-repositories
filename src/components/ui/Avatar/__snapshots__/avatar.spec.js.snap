// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should be match with snapshot 1`] = `
<DocumentFragment>
  <figure
    class=""
  >
    <a>
      <img
        alt="avatar"
        class="w-full h-auto rounded-full"
        height="296"
        width="296"
      />
    </a>
  </figure>
</DocumentFragment>
`;
